src/main/python/ayab/plugins/ayab_plugin/ayab_control.py

```diff
@@ -472,46 +537,46 @@ def knit(self, pImage, pOptions):
                     self.__logger.error("Could not open serial port")
                     result = AYABControlKnitResult.ERROR_SERIAL_PORT
  
-                self._current_state = KnittingState.INIT
+                self.__current_state = KnittingState.INIT
 
-        elif self._current_state == KnittingState.INIT:
+        elif self.__current_state == KnittingState.INIT:
             if rcvMsg == 'cnfInfo':
-                if rcvParam == self._API_VERSION:
-                    self._current_state = KnittingState.WAIT_FOR_INIT
+                if rcvParam == self.__API_VERSION:
+                    self.__current_state = KnittingState.WAIT_FOR_INIT
                     result = AYABControlKnitResult.WAIT_FOR_INIT
                 else:
                     self.__logger.error("wrong API version: " +
```

Comment:
Unrelated: Sooner or later, we should show the Firmware Version here, as the user is not aware of an API version at all.